workflow/controller/operator.go

```diff
@@ -1284,6 +1322,43 @@ func (woc *wfOperationCtx) assessNodeStatus(pod *apiv1.Pod, old *wfv1.NodeStatus
 	return nil
 }
 
+func (woc *wfOperationCtx) verifyOutputs(pod *apiv1.Pod, tmpl *wfv1.Template, node *wfv1.NodeStatus) error {
```

Comment:
I intended this to produce warnings in case the upgrade went weird - but should I remove it? It's a lot of extra code that I'd need to write tests for.